src/ng/directive/form.js

```diff
@@ -497,18 +507,27 @@ var formDirectiveFactory = function(isNgForm) {
                 event.preventDefault();
               };
 
+              var handleKeypress = function(event) {
```

Comment:
Actually, the whole submit handling seems problematic. In `ngForm`, we don't emulate the submit behavior of regular forms. So in order to be able to "submit" a detached form, we would need to emulate the submit behavior (as requested in https://github.com/angular/angular.js/issues/2513). The current patch doesn't achieve this, and without it, I am not sure how useful the detached form is.